src/Arcus.Messaging.Tests.Integration/Health/TcpHealthListenerTests.cs

```diff
@@ -34,7 +35,7 @@ public async Task TcpHealthServer_ProbeForHealthReport_ResponseHealthy()
                 using (var reader = new StreamReader(clientStream))
                 {
                     // Act
-                    string healthReport = await reader.ReadLineAsync();
+                    string healthReport = await reader.ReadToEndAsync();
```

Comment:
You had some problems with this?

Reply:
I was figuring out #21 so can be removed, but think it's always best to make sure we've read to the end, no? What if there's a new line in there for example, won't that break it?

Reply:
Yeah, right. Was just thinking that with an infinite/gigantic input; it could cause availability issues when there isn't any maximum of some sort.
Just thinking out loud; but this is a test so, less related I guess.

Reply:
It should be fine indeed I think!